### DISCUSSION TITLE
blog/the-blockchains-quantum-leap

### DISCUSSION THREAD

**Discussion by giscus** [rmourey26/BlocksquawkBlog#4] [Blog Comments]
# blog/the-blockchains-quantum-leap

What problems could the blockchain industry solve by leveraging their collective financial value to build pre and post quantum capable DAO owned data centers for the entire blockchain industry and to compete with cloud computing providers like AWS, Microsoft, and Google?

https://blocksquawk-blog.vercel.app/blog/the-blockchains-quantum-leap

**Comment by rmourey26** [comment]
blocksquawk test comment
